packages/utils/src/abi_encoder/evm_data_types/constructor.ts

```diff
@@ -0,0 +1,27 @@
+import { ConstructorAbi, DataItem, MethodAbi } from 'ethereum-types';
+
+import { DataType } from '../abstract_data_types/data_type';
+import { DataTypeFactory } from '../abstract_data_types/interfaces';
+import { DecodingRules, EncodingRules } from '../utils/rules';
+
+import { TupleDataType } from './tuple';
+
+export class ConstructorDataType {
```

Comment:
Could be good to inherit from `DataType` here. Both for consistency w other data types and so it can be used with `AbiEncoder.create()`